napalm_panos/panos.py

```diff
@@ -154,6 +154,12 @@ def _import_file(self, filename):
         else:
             return path
 
+    def is_alive(self):
+        if self.device:
```

Comment:
If the connection is established via SSH, you should probably return the connection status.